dmoj/graders/interactive.py

```diff
@@ -0,0 +1,40 @@
+from dmoj.graders.standard import StandardGrader
+
+class Interactor(object):
+    def __init__(self, process):
+        self.process = process
+
+    def read(self):
+        return self.process.stdout.read()
+
+    def readline(self):
```

Comment:
`readline` and `writeln`: one of these things is not like the other.

And it's `readline`.